shared/profile/render.desktop.js

```diff
@@ -14,33 +19,118 @@ class Render extends Component<void, Props, void> {
+      proofNotice = `Some of ${this.props.username}'s proofs have changed since you last tracked them.`
+    }
+
+    const folders = _.chain(this.props.tlfs)
+      .sortBy('isPublic')
+      .map(folder => (
+        <Box style={styleFolderLine}>
+          <Icon type={folder.isPublic ? 'icon-folder-public-32' : 'icon-folder-private-32'} style={styleFolderIcon} />
+          <Usernames users={folder.users} type={'Body'} />
+        </Box>
+      ))
+      .value()
+
     return (
-      <Box>
+      <Box style={styleContainer}>
         <Box style={{...styleHeader, backgroundColor: headerColor}} />
         <Box style={globalStyles.flexBoxRow}>
-          <UserBio
-            avatarSize={112}
-            username={this.props.username}
-            userInfo={this.props.userInfo}
-            currentlyFollowing={this.props.currentlyFollowing}
-            trackerState={this.props.trackerState}
-          />
-          <UserProofs
-            username={this.props.username}
-            proofs={this.props.proofs}
-            currentlyFollowing={this.props.currentlyFollowing}
-          />
+          <Box style={styleBioColumn}>
+            <UserBio
+              type='Profile'
+              avatarSize={112}
+              style={{marginTop: 39}}
+              username={this.props.username}
+              userInfo={this.props.userInfo}
+              currentlyFollowing={this.props.currentlyFollowing}
+              trackerState={this.props.trackerState}
+            />
+            <UserActions
+              style={styleActions}
+              trackerState={this.props.trackerState}
+              currentlyFollowing={this.props.currentlyFollowing}
+              onFollow={this.props.onFollow}
+              onUnfollow={this.props.onUnfollow}
+              onAcceptProofs={this.props.onAcceptProofs}
+            />
+          </Box>
+          <Box style={styleProofColumn}>
+            <Box style={styleProofNoticeBox}>
+              {proofNotice && <Text type='BodySmallSemibold' style={{color: globalColors.white}}>{proofNotice}</Text>}
+            </Box>
+            <UserProofs
+              style={styleProofs}
+              username={this.props.username}
+              proofs={this.props.proofs}
+              currentlyFollowing={this.props.currentlyFollowing}
+            />
+            {folders}
+          </Box>
         </Box>
       </Box>
     )
   }
 }
 
+const styleContainer = {
+  ...globalStyles.flexBoxColumn,
+  position: 'relative'
+}
+
 const styleHeader = {
-  flex: 1,
-  height: 96
+  position: 'absolute',
+  width: '100%',
+  height: HEADER_SIZE
+}
+
+const styleBioColumn = {
+  ...globalStyles.flexBoxColumn,
+  alignItems: 'center'
+}
+
+const styleActions = {
+  ...globalStyles.flexBoxRow,
+  marginTop: globalMargins.small
+}
+
+const styleProofColumn = {
+  ...globalStyles.flexBoxColumn,
+  width: 320,
+  marginLeft: globalMargins.medium,
+  marginRight: globalMargins.medium,
+  zIndex: 2
```

Comment:
Need to cover the absolutely-positioned header with the "Some proofs changed..." text, which is a child of this.